### DISCUSSION TITLE
Add ability to block new contacts on a alias based on the sender domain.

### DISCUSSION THREAD

**Discussion by netcarver** [simple-login/app#767] [Feature Requests]
@nguyenkims 

Hi Son, I have a couple of aliases which receive a mixture of wanted and unwanted messages. I can't just turn off the alias because it still gets a load of wanted/needed email.  Almost all of the unwanted messages come from randomised senders using one domain. 

Today I got a few emails from <random-sender>@spamming.domain so I'd like to have a feature allowing me to block contact creation on aliases based on the @spamming.domain used in the sender's email address.

Currently, I have to be reactive and go in to SimpleLogin multiple times a day and block these new contacts but tomorrow there will be more. I'd like to change that - all of these emails come from @spamming-domain and could be blocked in a rule-based way within SimpleLogin.  I know I could also add a rule in my email reader - but I'd rather do it within SimpleLogin to avoid eventually having thousands of bad contacts on an active alias.

Thanks for an excellent product!

**Comment by nguyenkims** [comment]
@netcarver thanks for the info, indeed it makes sense in this case to block a whole domain.
